interface/forms/gad7/view.php

```diff
@@ -16,11 +16,13 @@
 
 use OpenEMR\Common\Csrf\CsrfUtils;    // security module
 use OpenEMR\Core\Header;
-use Mpdf\Mpdf;  /* used to generate a pdf of the form */
+/* se Mpdf\Mpdf; /* to generate pdf files*/
```

Comment:
ok to remove

Reply:
yes please, thanks

Reply:
sorry, meant please go ahead and remove for us :)
also one lonely styling error
![Screenshot from 2022-04-20 14-33-16](https://user-images.githubusercontent.com/5618372/164299125-709957f4-a928-4a6f-8463-b4172a682632.png)